Cats/ViewControllers/StarWarsListVC.swift

```diff
@@ -0,0 +1,120 @@
+//
+//  StarWarsListVC.swift
+//  Cats
+//
+//  Created by Артем Соколовский on 07.07.2021.
+//
+
+import UIKit
+
+class StarWarsListVC: UIViewController, UITableViewDataSource, UITableViewDelegate, UISearchBarDelegate {
+    
+    @IBOutlet weak var table: UITableView!
```

Comment:
IBOutlets should be private to avoid leaking UIKit to higher layers ([Private Outlet](https://realm.github.io/SwiftLint/private_outlet.html))